### PR TITLE
Accept function without target in with constants statement

### PR DESCRIPTION
**Pull request by mrSundvoll**
Sets out to solve issue #34.

### CODE REVIEW

**Review by mrSundvoll — COMMENTED** [review]
Really simple fix that allows for simple expressions in with constants statements. Is this a too simple solution @JensGM ?

**Review by mrSundvoll — COMMENTED** [review]
Really simple fix that allows for simple expressions in with constants statements. Is this a too simple solution @JensGM ?

File: xun/tests/test_transformations.py [hunk 1]
```diff
@@ -218,3 +218,23 @@ def f():
...
+            procedure()
+        return 1    
+        
+    assert 'procedure' in workflow.dependencies
+    
+    result = workflow.blueprint().run(
+        driver=xun.functions.driver.Sequential(),
+        store=xun.functions.store.Memory(),
+    )
+    assert result == 1
```
Comment (JensGM) [inline xun/tests/test_transformations.py:240]: Missing new line at end of file

File: xun/functions/util.py [hunk 2]
```diff
@@ -420,6 +420,7 @@ def is_assignments_and_expressions(node):
     return all(
         isinstance(node, ast.Assign)
         or isinstance(node, ast.Expression)
+        or isinstance(node, ast.Expr)
         for node in node.body
     )
```
Comment (JensGM) [inline xun/functions/util.py:425]: It's possible to check all types at once. Also, I think the bug here was using `ast.Expression` in the first place, I _think_ it should've been `ast.Expr` all along

```suggestion
    return all(isinstance(node, (ast.Assign, ast.Expr)) for node in node.body)
```

**Review by JensGM — CHANGES_REQUESTED** [review]
I think both commits should be a single one. Ideally, all tests should pass for all commits in the history

**Review by JensGM — APPROVED** [review]
Looks good!

**Pull request merged**